bukkit/src/main/java/pl/spaceis/plugin/bukkit/BukkitSpaceIsCommand.java

```diff
@@ -25,33 +25,37 @@
 import org.bukkit.command.CommandSender;
 import pl.spaceis.plugin.config.Config;
 import pl.spaceis.plugin.config.EmptyConfigFieldException;
+import pl.spaceis.plugin.config.Messages;
+import pl.spaceis.plugin.resource.ResourceLoaderException;
 
-public class SpaceIsCommand implements CommandExecutor {
+public class BukkitSpaceIsCommand implements CommandExecutor {
```

Comment:
Skoro każda komenda robi to samo... Może by ją też jakoś zrobić abstrakcyjnie, żeby tylko się dostarczało implementacje np. hasPermission czy sendMessage